docs/ref/models/constraints.txt

```
@@ -130,7 +130,7 @@ ensures the age field is never less than 18.
 ``UniqueConstraint``
 ====================
 
-.. class:: UniqueConstraint(*expressions, fields=(), name=None, condition=None, deferrable=None, include=None, opclasses=(), nulls_distinct=None, violation_error_code=None, violation_error_message=None)
+.. class:: UniqueConstraint(*expressions, fields=(), name, condition=None, deferrable=None, include=None, opclasses=(), nulls_distinct=None, violation_error_code=None, violation_error_message=None)
```

A parameter without a default cannot be preceded by one with a default.
```suggestion
.. class:: UniqueConstraint(*expressions, name, fields=(), condition=None, deferrable=None, include=None, opclasses=(), nulls_distinct=None, violation_error_code=None, violation_error_message=None)
```

---

Ah, but since they're keyword-only, they can, no? You'd be quite right if it were `expressions` instead of `*expressions`.

---

TIL 😅